packages/autocomplete-presets/src/results.ts

```diff
@@ -24,10 +23,12 @@ export function getAlgoliaSource({
   if (typeof (searchClient as Client).addAlgoliaAgent === 'function') {
     if (__DEV__) {
       (searchClient as Client).addAlgoliaAgent(
-        `autocomplete.js (${version}-development)`
+        `autocomplete.js (${__VERSION__}-development)`
```

Comment:
I have a question at this point.
Babel did nothing with `import { version } from './package.json';`, so when `autocomplete-presets` is loaded at `autocomplete-react`, the resolution of `./package.json` failed.
I tried to look for any plugin to inline the version in the transpiled code, but coudn't find any.

Reply:
A (probably better) alternative would be to maintain a `version.ts` file [as we do in InstantSearch.js](https://github.com/algolia/instantsearch.js/blob/a2248b479d01104a2d7979c7dcfe937184d32a6f/src/lib/version.ts). We'll need to bump it in the release process though.

Reply:
@francoischalifour It makes sense. Let me ask you a question here. Do we want to go with independent versioning? or same version for all packages?

Reply:
Same version for all packages sounds easier to deal with.